subsys/net/lib/azure_iot_hub/src/azure_iot_hub_dps.c

```diff
@@ -16,7 +16,7 @@
 #include <azure/az_iot.h>
 
 #include "azure_iot_hub_dps_private.h"
-#include "azure_iot_hub_mqtt.h"
+#include "mqtt_helper.h"
```

Comment:
This should probably be `#include <mqtt_helper.h>` or perhaps `#include <net/mqtt_helper.h>`